framework/bucket_policies/s3_operation_access_tester.py

```diff
@@ -0,0 +1,50 @@
+from framework.bucket_policies.access_validation_strategy_factory import (
+    AccessValidationStrategyFactory,
+)
+from noobaa_sa.exceptions import UnexpectedBehaviour
+
+
+class S3OperationAccessTester:
```

Comment:
@sagihirshfeld Can't we skip writing new class and moving `check_client_access_to_bucket_op` method to bucket_utils and calling it from there